testutil/helpers.go

```diff
@@ -97,3 +98,8 @@ func LessLogging() (reset func()) {
 	logrus.SetLevel(logrus.ErrorLevel)
 	return func() { logrus.SetLevel(lvl) }
 }
+
+// PathToUri converts a file path to an URI. Compatible with both Windows and Linux paths.
+func PathToURI(p string) string {
+	return "file://" + filepath.ToSlash(p)
+}
```

Comment:
This function is only used twice in `external_match_test.go` . Since it's really one-line function, I think it's preferable to:
 - not use it at all, and use `file://` + filepath.ToSlash(p)` where we need.
 - if you prefer a function, you can create it, non exported, in `external_match_test.go`

In `list_test.go` this is used many times so the function is useful there. 

_A little copying is better than a little dependency._ https://go-proverbs.github.io/

Moreover, after that, the diff of the PR will get just get back to the `ExternalMatch` filter and its tests